packages/storybook/layouts.stories.js

```diff
@@ -0,0 +1,546 @@
+import { storiesOf } from '@storybook/polymer';
+import { withKnobs } from '@storybook/addon-knobs';
+import { html } from 'lit-html';
+
+import '@conversionxl/cxl-institute';
+
+storiesOf('Layouts', module)
+  .addDecorator(withKnobs)
+  .add('cxl-institute-layout (vaadin tabs nav)', () => {
+    return html`
+
+      <cxl-institute-layout>
+        <cxl-icon-nav slot="icon-nav">
+          <vaadin-tab>
+            <cxl-icon-nav-item href="#" content-type="icon">cxl:nav-trainings</cxl-icon-nav-item>
+          </vaadin-tab>
+          <vaadin-tab>
+            <cxl-icon-nav-item href="#" content-type="icon">cxl:nav-roadmap</cxl-icon-nav-item>
+          </vaadin-tab>
+          <vaadin-tab>
+            <cxl-icon-nav-item href="#" content-type="icon">cxl:nav-discover</cxl-icon-nav-item>
+          </vaadin-tab>
+          <vaadin-tab theme="cxl-icon-nav-item-bottom">
+            <cxl-icon-nav-item href="#" content-type="icon">lumo:user</cxl-icon-nav-item>
+          </vaadin-tab>
+          <vaadin-tab theme="cxl-icon-nav-item-button">
+            <vaadin-button theme="primary">
+              Start course
+            </vaadin-button>
+          </vaadin-tab>
+        </cxl-icon-nav>
+
+        <!-- Sidebar -->
+        <h3 slot="sidebar-header" part="sidebar-header">Training Manager</h3>
+        <div slot="sidebar-content">
+          <strong class="extra-heading">Digital analytics minidegree program</strong>
+          <br><br>
+          <cxl-vaadin-accordion>
+            <vaadin-accordion-panel theme="cxl-sidebar-panel">
+              <div slot="summary">
+                <vaadin-item theme="cxl-sidebar-panel-summary">
+                  <label class="label">Track #1</label>Google Analytics
```

Comment:
@chanar `<label class="label">`... uh why class this?

Reply:
"Google Analytics" title is missing a header. `<h4 class="entry__title">` to start.